plugins/sim_gui/CMakeLists.txt

```diff
@@ -7,10 +7,10 @@ add_library(${plugin_lower} SHARED ${sources})
 target_link_libraries(${plugin_lower}
   PRIVATE
   ${PROJECT_LIBRARY_TARGET_NAME}
-  gz-sim${GZ_SIM_VER}
-  gz-sim${GZ_SIM_VER}::gui
-  gz-sim${GZ_SIM_VER}::gz-sim${GZ_SIM_VER}
-  gz-gui${GZ_GUI_VER}::gz-gui${GZ_GUI_VER}
```

Comment:
oops, I'll revert this line

Reply:
https://github.com/gazebosim/gz-launch/pull/292/commits/4966abf01110169dfb893e08c9c968a60f7261d4